modules/op-understanding-credential-selection.adoc

```diff
@@ -0,0 +1,42 @@
+// This module is included in the following assembly:
```

Comment:
Readers LOVE examples like the ones you've provided here! In web analytics for technical documentation, "examples" is one of the most frequently used search terms.